plugins/CoreHome/CoreHome.php

```diff
@@ -487,4 +489,19 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'General_PreviousPeriod';
         $translationKeys[] = 'General_PreviousYear';
     }
+
+    /**
+     * Ensures dimensions that should have been removed are not used any longer even if the file failed to remove on update
+     *
+     * @param $instances
+     */
+    public function removeOldDimensions(&$instances)
+    {
+        $removedDimensions = Dimension::getRemovedDimensions();
+
+        $instances = array_filter($instances, function ($dimension) use ($removedDimensions) {
+            $className = get_class($dimension);
```

Comment:
btw I suppose there's no performance difference between `get_class` vs `$dimension::class`?  Might not be needed though this improvement anyway as it's only called like 100 times

Reply:
`::class` doesn't work on instances. Also get_class returns kind of the "true" names of classes, whereas `::class` returns the name used. See this example:
```
echo get_class(new DatETime()); // prints DateTime
echo DatETime::class; // prints DatETime
```